crates/citrea-stf/src/verifier.rs

```diff
@@ -37,7 +37,7 @@ where
         zkvm: Zk,
         pre_state: Stf::PreState,
     ) -> Result<(), Da::Error> {
-        println!("Running sequencer commitments in DA slot");
+        // println!("Running sequencer commitments in DA slot");
```

Comment:
That's what Esad asked not to remove. Please add a commend above "Don't remove"

Reply:
fixed